### DISCUSSION TITLE
There should be a Beta Wiki for Osu!Lazer Editor

### DISCUSSION THREAD

**Discussion by CuredDDN** [ppy/osu#27233] [Ideas] [answered]
**Issue**
After the lazer pp update, there is now an option on the osu! website to change into lazer mode. Now I've switched to lazer for around two months now, and I've been trying out the beatmap editor recently. I love the beatmap editor by the way, and I know it's unfinished. 

I wanted to find information about how it works on the osu!wiki at index > Beatmap Editor. But currently, it's only shows info about stable! So I thought how to make suggestions about how it could work and if it could be added, even though the editor is not finished.

**Ideas**
The Beatmap Editor wiki changes when Lazer Mode gets turn on to show information about the lazer editor, simple right? But currently, the editor is still changing, and changing all that text every time when something about the editor is updated is a hassle. So a temporary solution is to add a disclaimer at the top of the wiki that states something similar to this, "This wiki was published during an older version of osu!lazer, some information might be outdated." etc. etc. This is so that people can find information on how the editor works, and works with the changing editor, aka Beta as these aren't the final versions.

This could be applied to other wiki's, but i'm more focused on the editor.

**Comment by bdach** [comment, marked as answer]
It's too early. I don't consider the editor to be in a ready state as of now. Not until beatmap submission is implemented at least.
